controllers/admin/AdminImagesController.php

```diff
@@ -393,6 +417,26 @@ public function initModal(): void
         ];
     }
 
+    public function updateOptionPsImageFormat($value): void
+    {
+        if ($this->access('edit') != '1') {
+            throw new PrestaShopException($this->trans('You do not have permission to edit this.', [], 'Admin.Notifications.Error'));
+        }
+
+        if ($this->isMultipleImageFormatFeatureEnabled && !$this->errors && $value) {
+            Configuration::updateValue('PS_ADDITIONAL_IMAGE_JPG', in_array('jpg', $value));
```

Comment:
I don't understand why you used four different configuration keys, if its a list of select formats shouldn't this be a single `PS_IMAGE_FORMATS` configuration containing imploded values instead? Like `PS_IMAGE_FORMATS=jpg,webp,avif` for example?